pennylane_lightning/src/benchmarks/Bench_BitUtil.cpp

```diff
@@ -11,6 +11,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+#include <bit>
```

Comment:
No need for this one if it's already included in `BitUtil.hpp` I guess.
```suggestion
```

Reply:
As above